src/main/java/entry/Task.java

```diff
@@ -0,0 +1,54 @@
+package entry;
+
+public class Task {
```

Comment:
Add a header comment to describe the use of this task.
I think can declare Task as an abstract class since there is no Task instance.